database/tables/workspace_logs.pg

```diff
@@ -3,8 +3,8 @@ columns
     id: bigint not null default nextval('workspace_logs_id_seq'::regclass)
     level: enum_log_level
```

Comment:
Are we ever using a log level other than `info`? If not, I'm not sure we need a column for `level` at all?

Reply:
Currently we're not, but I thought we were planning to log errors eventually?